docs/community_resources/phpstorm_plugin.md

```diff
@@ -0,0 +1,171 @@
+# Ibexa DXP plugin for PhpStorm
+
+Ibexa DXP plugin for PhpStorm helps you to work with Ibexa DXP by speeding up installation
+and providing file templates, intentions, autocompletion, and other features.
+
+## Requirements
+
+- PhpStorm 2021.1.1 or newer
+- Enabled Symfony support plugin
+
+## Install PhpStorm plugin
+
+You can install the Ibexa DXP plugin for PhpStorm from the JetBrains Marketplace,
+or manually, from a downloaded .jar file.
+
+### Install from JetBrains Marketplace
+
+To install plugin from JetBrains marketplace:
+
+1\. In PhpStorm settings, select **Plugins** > (gear icon) > **Manage Plugin Repositories**.
+
+![Install Ibexa DXP PhpStorm plugin from the marketplace](img/phpstorm_plugin_installation.png)
+
+2\. Add `https://plugins.jetbrains.com/plugins/eap/17239` as new plugin repository.
+
+3\. Look for "Ibexa DXP" in the plugin browser and click **Install**.
+
+### Install from file
+
+You can also install the plugin manually from a `.jar` file.
+To do so, download the latest version of the plugin from [JetBrains Marketplace](https://plugins.jetbrains.com/plugin/17239-ibexa-dxp/versions).
+Then, in PhpStorm settings. select **Plugins** > (gear icon) > **Install plugin from Disk...**
```

Comment:
If it is possible, I would add a gear icon image too.

Reply:
Unfortunately, mkdocs does not play well with inline images

Reply:
Maybe we can use "gear" emoji character? https://www.utf8icons.com/character/9881/gear

Reply:
This is what an inline image would look like: 
![image](https://user-images.githubusercontent.com/13621049/126763447-705026c7-ec8e-4276-bcec-3a5ab80b742b.png)
while the emoji is not really similar to the default icon in Storm and would be more confusing than helpful